pkg/kubeapiserver/server/insecure_handler.go

```diff
@@ -78,8 +78,8 @@ func (s *InsecureServingInfo) NewLoopbackClientConfig(token string) (*rest.Confi
 		// Increase QPS limits. The client is currently passed to all admission plugins,
 		// and those can be throttled in case of higher load on apiserver - see #22340 and #22422
 		// for more details. Once #22422 is fixed, we may want to remove it.
-		QPS:   50,
-		Burst: 100,
+		QPS:   60,
```

Comment:
This is on the InsecureServingInfo type-- is this the wrong one or do we really run the loopback client on the insecure port?

Reply:
Side note: this is a surprising place to have what amounts to a global QPS limit.

Why make this a few percent bigger instead of 10x or .1x?

Reply:
It seems like the "right" way to do this is to track, per-handler, a running average of the "fan out", and then weight each handler accordingly. It's very weird that you could run out of QPS while partway through servicing a call.

Reply:
https://www.google.com/search?q=why+do+we+even+have+that+lever